### DISCUSSION TITLE
Live Timing: speeds in timing window or separate

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#197] [Feature requests] [answered]
I would like to have the speeds from the sectors and the speed trap in the timing window.
Maybe, if it would look too cluttered, then as a separate window.

**Comment by jeppeolesen** [comment, marked as answer]
MultiViewer's live timing now includes both the speed trap and sector times leaderboards.

![Screenshot 2025-03-23 at 00 39 24](https://github.com/user-attachments/assets/ed953714-41a5-414e-888d-507f5060fc79)
![Screenshot 2025-03-23 at 00 39 30](https://github.com/user-attachments/assets/7b69ca8c-396b-451c-935b-42b88f83e4f8)
